boards/BMS/Test/Inc/Bms_Test.h

```diff
@@ -0,0 +1,26 @@
+#pragma once
```

Comment:
If we left the class definition in `Test/Src/Bms_Test.cpp`, the build system would get confused because it treats each `.cpp` (that is not `main.cpp`) as its own separate, compilable Gtest binary. I don't imagine this file getting too long anyway so I think leaving it header-only is fine for now.

Reply:
Uhhhh, we should probably not be doing it like that. In general, we should compile all the `test` files into a single executable, and having a single (shared) `gtest_main.cpp` file that has the `main` function in it. Does that make sense?

Looks like you don't even need to create `gtest_main.cpp`, gtest provides one for you: https://stackoverflow.com/a/21479008

Reply:
Oh ok yeah I get what you mean. Can we have one `main` function per board? Or did you mean one `main` function for every board (i.e. Entire project)?

Reply:
I went ahead and created one test executable per board. All the `main.cpp` are purged as well.

Reply:
Yep, that's what I was thinking of.